extensions/kafka/src/main/java/io/deephaven/processor/ObjectProcessorRowLimited.java

```diff
@@ -12,9 +12,8 @@
  * {@link #processAll(ObjectChunk, List)}.
  *
  * <p>
- * For use-cases that are naturally performed one row at a time, callers are encouraged to extend
- * {@link ObjectProcessorRowBase}. For use-cases where the implementation can be columnar but callers would like to add
- * a row limit, callers are encouraged to create a columnar implementation against {@link ObjectProcessor} and use
+ * For use-cases where the implementation can be columnar but callers would like to add a row limit, callers are
```

Comment:
This comment looks orphaned without the previous sentence. What about row-oriented use cases? Maybe delete the whole thing?

Reply:
I think it's still relevant. Essentially, it provides guidance around "should my implementation extend ObjectProcessorRowLimited?". If an implementation _can_ be column oriented, but you want to add a row limit for reasons outlined, you'll be encouraged to still create the column oriented version, but call into `ObjectProcessorRowLimited#of` instead of extending `ObjectProcessorRowLimited`..

Reply:
We discussed getting rid of the interface and squashing it into the impl. Should clarify that this is not a meaningful thing to do for row-oriented impls.